packages/business-rules/src/rules/appeal/deadline-date.js

```diff
@@ -46,7 +49,16 @@ module.exports = (
 			? appeal.type[type].appealDue.time
 			: appeal.type[type].appealDue[decision].time;
 
-	return add(endOfDay(decisionDate), {
-		[duration]: time
-	});
+	// given a utc datetime get the UK time
+	const decisionDateUK = utcToZonedTime(decisionDate, targetTimezone);
```

Comment:
Willing to say I've overthought it or have faulty tests, however, doing that would break the current tests based on the timezone they are run in

I _believe_ the way I've currently done it, it gets the end of the day GMT regardless of which timezone the computer running the get deadline check is in and returns equivalent utc time